lib/src/definitions.dart

```diff
@@ -71,17 +71,21 @@ class MapViewConfiguration {
     required this.situmApiKey,
     this.buildingIdentifier,
     this.remoteIdentifier,
-    this.viewerDomain = "https://map-viewer.situm.com",
+    this.viewerDomain = "map-viewer.situm.com",
     this.apiDomain = "dashboard.situm.com",
     this.directionality = TextDirection.ltr,
     this.enableDebugging = false,
   });
 
   String get _internalViewerDomain {
-    if (viewerDomain.endsWith("/")) {
-      return viewerDomain.substring(0, viewerDomain.length - 1);
+    String finalViewerDomain = viewerDomain;
+    if (!finalViewerDomain.startsWith("https://")) {
```

Comment:
Esto igual hace algo raro si alguien pone http en vez de https

Reply:
Sip, hay que comentar este if en caso de hacer pruebas en local. Puede llegar a necesitar algun cliente "http://", entiendo que no cierto?

Reply:
Non teria máis sentido meter no if se non empeza nin por https nin por http, asi non tes que andar facendo cambios en modo local?